test/rest_api/test_storage_service.py

```diff
@@ -139,6 +139,14 @@ def test_storage_service_keyspace_scrub(cql, this_dc, rest_api):
                 resp = rest_api.send("POST", f"storage_service/keyspace_scrub/{keyspace}", { "cf": f"{test_tables[0]},XXX" })
                 assert resp.status_code == requests.codes.not_found
 
+def test_storage_service_keyspace_scrub_abort(cql, this_dc, rest_api):
+    with new_test_keyspace(cql, f"WITH REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 1 }}") as keyspace:
+        with new_test_table(cql, keyspace, "a int, PRIMARY KEY (a)") as t0:
+            with scylla_inject_error(rest_api, "rest_api_keyspace_scrub_abort"):
+                cql.execute(f"INSERT INTO {t0} (a) VALUES (42)")
```

Comment:
Don't we need to flush the table first? Otherwise there will be no sstables to scrub.